jwst/flatfield/tests/test_combine_fast_slow.py

```diff
@@ -34,6 +34,7 @@ def test_combine_fast_slow():
              'c5': 2.0e-3}
     poly = polynomial.Polynomial1D(degree=5, **coeff)
     tab_flat = poly(tab_wl)
+    tab_flat_err = np.full(tab_flat.shape, np.nan, dtype=tab_flat.dtype)
```

Comment:
I wasn't quite sure what values would be suitable for testing the error interpolation. I picked an array of `nan` as that is the expected "default" value that the datamodel will generate if a reference file is opened that does not contain an `error` column.